library/core/resource-loader/src/main/java/org/quiltmc/qsl/resource/loader/api/package-info.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * The Resource Loader.
+ *
+ * <p><h3>Quick note about vocabulary in Resource Loader and Minecraft:</h3>
+ * <ul>
+ *  <li>Resource Pack refers to both client-sided resource pack and data pack.</li>
+ *  <li>Virtual Resource Pack refers to a resource pack that may be generated at runtime, or simply doesn't exist directly on disk.</li>
+ *  <li>Group Resource Pack refers to a virtual resource pack that groups multiple resource packs together.</li>
+ * </ul>
+ * </p>
+ *
+ * <p><h3>Modded Resource Pack Handling</h3></p>
+ * <p>The Resource Loader will create a resource pack for each mod that provides resources in {@code assets} or {@code data}
+ * sub-directories.
+ * Those mod resource packs are grouped into the default resource pack, they can override the default resource pack but unrecommended.</p>
+ *
+ * <p><h4>Built-in Mod Resource Pack</h4></p>
+ * <p>The Resource Loader adds manually registered mod resource packs. Those resource packs are registered with
+ * {@link org.quiltmc.qsl.resource.loader.api.ResourceLoader#registerBuiltinResourcePack(net.minecraft.util.Identifier, net.fabricmc.loader.api.ModContainer, ResourcePackActivationType)}</p>
+ *
+ * <p><h4>Programmer Art Resource Pack</h4></p>
+ * <p>The Resource Loader will inject resources into the Programmer Art resource pack for each mod that provides
+ * Programmer Art resources in the {@code programmer_art} top-level directory of the mod
```

Comment:
You refer to what I am presuming is the same location (the subfolders you create in the `resources` folder) as being both a sub-directory and a top-level directory.

Reply:
Now it's top-level directory everywhere.